### PR TITLE
Update kubernetes version to 1.13.0 alpha.3

### PR DESCRIPTION
**Pull request by feiskyer**
This PR

- Update kubernetes version to 1.13.0 alpha.3
- Update acs-engine to v0.25.0
- Make their version configurable by env variables

### CODE REVIEW

**Comment by feiskyer** [comment]
@justaugustus v1.13.0-beta.0 has been released yet because of blocking test failures. See https://groups.google.com/forum/#!topic/kubernetes-dev/hWhK6lCJK9s.

File: tests/k8s-azure/manifest/linux.json [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
+            "count": 1,
+            "dnsPrefix": "{dnsPrefix}",
+            "vmSize": "Standard_F2"
+        },
+        "agentPoolProfiles": [
+            {
+                "name": "agentpool1",
+                "count": 2,
+                "vmSize": "Standard_F2",
+                "availabilityProfile": "VirtualMachineScaleSets",
```
Comment (karataliu) [inline tests/k8s-azure/manifest/linux.json]: We can add another test to verify vmss, for here we still need to test vms.

File: tests/k8s-azure/k8s-azure [hunk 2]
```diff
@@ -63,7 +64,7 @@ sub deploy {
                 }, undef, $ENGINE_FILE)
               && do {
                 if ($accm_image) {edit_file($ENGINE_FILE, sub { s/"customCcmImage":\h*"\K.*"/$accm_image"/ }); }
-                else { 1; }
+                if ($hyperkube_image) {edit_file($ENGINE_FILE, sub { s/"customHyperkubeImage":\h*"\K.*"/$hyperkube_image"/ }); }
```
[the remark this answers is not in the document]
Comment (feiskyer) [reply tests/k8s-azure/k8s-azure]: ACK, fixed in the new comments.

File: tests/k8s-azure/Dockerfile [hunk 3]
```diff
@@ -10,10 +10,9 @@ RUN git clone https://github.com/kubernetes/kubernetes . \
     && make ginkgo \
     && go build hack/e2e.go \
     && ./e2e
-RUN rm -rf _output/local/go
```
[the remark this answers is not in the document]
Comment (feiskyer) [reply tests/k8s-azure/Dockerfile]: Oops, removed accidentally.

File: tests/k8s-azure/manifest/linux.json (hunk 1 above)
Comment (feiskyer) [reply tests/k8s-azure/manifest/linux.json]: Good catch. This is from original PR, forgot to rollback.

**Comment by feiskyer** [comment]
@karataliu Addressed comments. PTAL

**Comment by karataliu** [comment]
/lgtm

'alpha.*' is on master branch, while 'beta.*' is on release branch. 
For PRs merged to master branch here, we'd sync to 'alpha.*' tags.
https://github.com/kubernetes/cloud-provider-azure/blob/master/docs/release-versioning.md

**Pull request merged**